semeio/jobs/correlated_observations_scaling/exceptions.py

```diff
@@ -6,3 +6,9 @@ class ValidationError(ValueError):
     def __init__(self, message, errors):
         super(ValidationError, self).__init__(message)
         self.errors = errors
+
+    def __str__(self):
+        msg = "{}\n".format(str(super(self)))
```

Comment:
This gives some slightly non standard behavior, so can´t for example check the error message when running pytest in the standard way:

```
with pytest.raises(ValueError) as err_msg:
    raise ValueError("my error")
```
Then `str(err_msg.value)` should give: `my error`, but with ValidationError it throws an error.